checker/build.gradle

```diff
@@ -4,7 +4,7 @@ ext {
     // See instructions for updating jdk8.jar at
     // https://github.com/typetools/annotated-libraries/blob/master/README.md .
     // Set jdkShaHash to 'local' to use a locally-built version of jdk8.jar .
-    jdkShaHash = '59a70f00fadc6abf888e8b3e98430792e4db617a'
+    jdkShaHash = '07a52880879073ad3caa5e39f21b0ba2d48c5eab'
```

Comment:
To make the continuous integration tests pass, could you please update the annotated JDK?  Thanks.

Reply:
Done!